web/views.py

```
@@ -5285,6 +5286,175 @@ def toggle_course_status(request, slug):
     return redirect("course_detail", slug=slug)
 
 
+# NFT management views
+
+
+@login_required
+@teacher_required
+def send_nft_badge(request, achievement_id):
```

Ruff (0.8.2)

`send_nft_badge` is too complex (12 > 10)

(C901)